services/api/src/admin_org.py

```diff
@@ -32,33 +52,34 @@ def get_all_orgs():
     return return_obj
 
 
-def get_org_data(org_name):
+def get_org_data(org_name: str, isAdminInOrg: bool):
```

Comment:
polish(non-blocking): use standard naming convention
```suggestion
def get_org_data(org_name: str, is_admin_in_org: booll):
```

Reply:
done